packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -452,14 +452,16 @@ contract DeployImplementations is Script {
     // --- Fault Proofs Contracts ---
 
     // The fault proofs contracts are configured as follows:
-    //   - DisputeGameFactory: Proxied, bespoke per chain.
-    //   - AnchorStateRegistry: Proxied, bespoke per chain.
-    //   - FaultDisputeGame: Not proxied, bespoke per chain.
-    //   - PermissionedDisputeGame: Not proxied, bespoke per chain.
-    //   - DelayedWETH: Proxied, and two bespoke ones per chain (one for each DisputeGame).
-    //   - PreimageOracle: Not proxied, shared by all standard chains.
-    //   - MIPS: Not proxied, shared by all standard chains.
-    //   - OptimismPortal2: Proxied, shared by all standard chains.
+    // | Contract                | Proxied | Deployment                        | MCP Ready  |
+    // |-------------------------|---------|-----------------------------------|------------|
+    // | DisputeGameFactory      | Yes     | Bespoke                           | Yes        |
+    // | AnchorStateRegistry     | Yes     | Bespoke                           | No         |
+    // | FaultDisputeGame        | No      | Bespoke                           | No         |
+    // | PermissionedDisputeGame | No      | Bespoke                           | No         |
+    // | DelayedWETH             | Yes     | Two bespoke (one per DisputeGame) | No         |
```

Comment:
DelayedWETH is MCP ready iff all standard chains are required to have the same `DELAY_SECONDS` immutable. This is currently unspecified by the standard config, but I believe we can assume it should be the same. I've asked @pauldowman to get the standard config updated here: https://discord.com/channels/1244729134312198194/1275532049180131429/1285623995789738004